Multiple scripts fail due to curl missing

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

Caddy, Ntfy, and probably more

### 📂 What was the exact command used to execute the script?

scripts-cli launch ntfy ntfy-test

### 📝 Provide a clear and concise description of the issue.

It seems that the images Incus provides might be lighter than those used on Proxmox and some dependencies such as `curl` (and probably other dependencies) are assumed to be installed when they are not. This can be seen with Ntfy and Caddy (what I have found so far)

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

_No response_

### 🔄 Steps to reproduce the issue.

* Try and launch either a Ntfy or Caddy instance
* See it error when installing the application due to `curl` missing

### ❌ Paste the full error output (if available).

```
Creating ntfy-test
  ✔  Set up Container OS
  ✔  Network Connected: 10.243.146.140 fd42:ef42:57f1:5ba2:216:3eff:feca:48a9 
  ✔  IPv4 Internet Connected
  ✖  IPv6 Internet Not Connected
  ✔  DNS Resolved github.com to 140.82.114.4
  ✔  Updated Container OS
  ✔  Installed Dependencies
 ⠋bash: line 23: curl: command not found
gpg: no valid OpenPGP data found.

[ERROR] in line 23: exit code 0: while executing command gpg --dearmor -o /etc/apt/keyrings/archive.heckel.io.gpg

   Instance Details                                                               
                                                                                  
  • Instance Name: ntfy-test                                                      
  • Application: ntfy                                                             
  • Image: images:debian/12                                                       
  • Incus Profiles: default                                                       
                                                                                  
  ## Application Information                                                      
                                                                                  
  ntfy (pronounced notify) is a simple HTTP-based pub-sub notification service. It
  allows you to send notifications to your phone or desktop via scripts from any  
  computer, and/or using a REST API. It's infinitely flexible, and 100% free      
  software.                                                                       
                                                                                  
  ## Resources                                                                    
                                                                                  
  Website: ntfy https://ntfy.sh/                                                  
                                                                                  
  Documentation: ntfy                                                             
                                                                                  
  Application Port : 80                                                           
```

### 🖼️ Additional context (optional).

_No response_

> Hi just wanted to let you know that this seems to still be happening on some of them.
> 
> I tried specifically with runtipi and cosmos on Debian 12
> 
> Issue was able to be bypassed by entering the terminal in incus of the container it creates, installing curl, then triggering the script inside the container.

Able to reproduce:
```
scripts-cli launch runtipi test-cli
Creating test-cli
  ✔  Set up Container OS
  ✔  Network Connected: 10.243.146.111 fd42:ef42:57f1:5ba2:1266:6aff:fe3e:f851 
  ✔  IPv4 Internet Connected
  ✖  IPv6 Internet Not Connected
  ✔  DNS Resolved github.com to 140.82.113.3
  ✔  Updated Container OS
 ⠋bash: line 21: curl: command not found

[ERROR] in line 21: exit code 0: while executing command curl -fsSL "https://raw.githubusercontent.com/runtipi/runtipi/master/scripts/install.sh" -o $(basename "https://raw.githubusercontent.com/runtipi/runtipi/master/scripts/install.sh")
```

--- a/codemods/update_os.sh
+++ b/codemods/update_os.sh
@@ -15,7 +15,7 @@ EOF
   $STD apt-get update
   $STD apt-get -o Dpkg::Options::="--force-confold" -y dist-upgrade
   rm -rf /usr/lib/python3.*/EXTERNALLY-MANAGED
-  $STD apt-get install -y wget logrotate
+  $STD apt-get install -y wget logrotate curl
   if [[ "$INSTALL_SSH" == "yes" ]]; then
     $STD apt-get install -y openssh-server
   fi

